Description: Write test file in /tmp
Last-Update: 2023-02-10
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/tests/test.rs
+++ b/tests/test.rs
@@ -39,6 +39,7 @@
         "defaultvalues"
     );
     config.setstr("DEFAULT", "defaultvalues", None);
+    std::env::set_current_dir("/tmp")?;
     config.write("output.ini")?;
     let map2 = config.clone().load("output.ini")?;
     assert_eq!(map2, *config.get_map_ref());
@@ -141,6 +142,7 @@
         "defaultvalues"
     );
     config.setstr("default", "defaultvalues", None);
+    std::env::set_current_dir("/tmp")?;
     config.write("output2.ini")?;
     let map2 = config.clone().load("output2.ini")?;
     assert_eq!(map2, *config.get_map_ref());
@@ -258,6 +260,7 @@
 
     let mut config = Ini::new();
     config.read(OUT_FILE_CONTENTS.to_owned())?;
+    std::env::set_current_dir("/tmp")?;
     config.write("output_sync.ini")?;
 
     async_std::task::block_on::<_, Result<_, String>>(async {
